docs/src/use/command-line-interface.md

```diff
@@ -67,36 +67,33 @@ You can view all the CLI options by running `npx eslint -h`.
 eslint [options] file.js [file.js] [dir]
 
 Basic configuration:
-  --no-eslintrc                   Disable use of configuration from .eslintrc.*
-  -c, --config path::String       Use this configuration, overriding .eslintrc.* config options if present
-  --env [String]                  Specify environments
-  --ext [String]                  Specify JavaScript file extensions
```

Comment:
Is it intended that `--no-eslintrc`, `--env` and `--ext` are no longer listed in this section, but still have dedicated paragraphs below?

Reply:
Yes. We are switching to show the default output everyone sees, but keeping the individual descriptions available so people can still look up the older options.